pom.xml

```diff
@@ -2715,6 +2715,27 @@
               <!-- spotless manve plugin does not allow empty here, so use \n -->
               <replacement>\n</replacement>
             </replaceRegex>
+            <!--
+              e.g., rewrite
+              /** @return blabla */
+              or
+              /**
+               * @return blabla
+               */
+              to
+              /** Returns blabla */
+              See https://errorprone.info/bugpattern/FutureReturnValueIgnored
+            -->
+            <replaceRegex>
+              <name>Purge single returns tag multi line</name>
+              <searchRegex>(?m)^ */\*\*\n *\* *@return *(.*) *\n *\*/$</searchRegex>
+              <replacement>/** Returns $1 */</replacement>
+            </replaceRegex>
+            <replaceRegex>
+              <name>Purge single returns tag single line</name>
+              <searchRegex>^ */\*\* *@return *(.*) *\*/$</searchRegex>
+              <replacement>/** Returns $1 */</replacement>
+            </replaceRegex>
             <!-- apply a specific flavor -->
```

Comment:
Is 2.23.0 bump required for specifying this regex or both (version bump + regex) are independent changes are we are combining for good?

Reply:
Waiting for this question to be resolved before merging. I merged a PR that causes a conflict in FavoredNodesPlan. Happy to fix it up during the merge.

Reply:
The version bumpiong is just a nice to have, not a must. If you think we'd better do it in a separated issue, we could open another one, not a big deal.

Reply:
Sounds good, I think combining with this PR was also fine, I just wanted to know whether this regex was going to be effective with version bump. But now that this PR is merged, let's bump it now with HBASE-27221. Thanks for this nice work @Apache9